anycloud/cli/src/deploy/mod.rs

```diff
@@ -543,6 +549,52 @@ pub async fn add_deploy_config() {
       break;
     }
   }
+  let prompt = if creds.len() > 1 {
+    "Minimum number of replicas per region or cloud"
```

Comment:
Nit: I don't know if "replicas" is how the end user is going to think of this. Maybe just `s/replicas/VMs/g` for all of the user-facing strings?

Reply:
sounds good